docs/configuring-playbook.md

```diff
@@ -33,6 +33,7 @@ When you're done with all the configuration you'd like to do, continue with [Ins
 
 - [Setting up the Jitsi video-conferencing platform](configuring-playbook-jitsi.md) (optional)
 
+- [Setting up budget builds or resource-constrained builds](configuring-playbook-budget-builds.md) (optional)
```

Comment:
Maybe renaming this to `configuring-playbook-dynamic-dns.md` is better for the Dynamic DNS feature (which may be usable for others as well)?

Or, we could have 2 docs pages linking to each other.

Reply:
Good call! I've rebranded this article